### PR TITLE
Expose the avatar url on users

### PR DESCRIPTION
**Pull request by steftrenson**

### CODE REVIEW

**Review by luisaterrabrito — DISMISSED** [review]

File: src/01-general/users.apib [hunk 1]
```diff
@@ -40,6 +40,7 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
+            + avatar_url: `www.teamleder.eu/your-avatar` (string)
```
Comment (luisaterrabrito) [inline src/01-general/users.apib]:
```suggestion
            + avatar_url: `www.teamleader.eu/your-avatar` (string)
```

**Review by luisaterrabrito — DISMISSED** [review]

File: src/01-general/users.apib [hunk 2]
```diff
@@ -40,6 +40,7 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
+            + avatar_url: `www.teamleader.eu/your-avatar` (string)
```
Comment (stivni) [inline src/01-general/users.apib]: PP: 
```suggestion
            + avatar_url: `https://www.teamleader.eu/your-avatar` (string)
```

File: src/01-general/users.apib [hunk 3]
```diff
@@ -88,6 +89,7 @@ Get a list of all users.
                     + Members
                         + active
                         + deactivated
+                + avatar_url: `www.teamleader.eu/your-avatar` (string)
```
Comment (stivni) [inline src/01-general/users.apib]:
```suggestion
                + avatar_url: `https://www.teamleader.eu/your-avatar` (string)
```

File: src/01-general/users.apib [hunk 4]
```diff
@@ -132,3 +134,4 @@ Get details for a single user.
                 + Members
                     + active
                     + deactivated
+            + avatar_url: `www.teamleader.eu/your-avatar` (string)
```
Comment (stivni) [inline src/01-general/users.apib]:
```suggestion
            + avatar_url: `https://www.teamleader.eu/your-avatar` (string)
```

**Review by stivni — DISMISSED** [review]
but I'd like the url to change to the correct path... Not sure what that's going to be though...

eg. `https://app.teamleader.eu/avatars/08765434567-aed876543ed.png` or something

**Review by jnstr — DISMISSED** [review]

File: apiary.apib [hunk 5]
```diff
@@ -699,7 +699,7 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
-            + avatar_url: `www.teamleder.eu/your-avatar` (string)
```
Comment (adognameddog) [inline apiary.apib]: I bought my sofa at Teamleder

**Review by adognameddog — DISMISSED** [review]

**Review by JorenSaey — APPROVED** [review]

**Review by luisaterrabrito — APPROVED** [review]

File: src/01-general/users.apib [hunk 6]
```diff
@@ -40,6 +40,7 @@ Get the current authenticated user.
             + time_zone: `Europe/Brussels` (string)
             + preferences (object)
                 + invoiceable: true (boolean)
+            + avatar_url: `https://www.teamleader.eu/your-avatar` (string, nullable)
```
Comment (jenssegers) [inline src/01-general/users.apib:43]: PP `avatar`, but no real strong opinion.

**Review by jenssegers — APPROVED** [review]

**Review by tiagobastos — APPROVED** [review]

**Review by psrpinto — APPROVED** [review]

**Comment by tiagobastos** [comment]
Is this going somewhere or can we close it?

**Comment by lowiebenoot** [comment]
The avatar url is exposed in the meanwhile, so I guess this still has to be merged?

**Comment by jenssegers** [comment]
Yes, then it will be gone from my "reviewed prs" tab 😄

**Pull request closed without merging**